.github/workflows/release.yml

```diff
@@ -16,42 +16,14 @@ jobs:
   main:
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout repo
-        uses: actions/checkout@v3
-      - name: Setup node
-        uses: actions/setup-node@v3
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
         with:
           cache: 'yarn'
-      - name: Install deps
-        # this runs a build script so there is no dedicated build
-        run: yarn install
-      - name: Run storybook build script
-        run: yarn build-storybook
-  release:
-    needs: main
-    runs-on: ubuntu-latest
-    if: ${{ github.repository == 'pmndrs/drei' && contains('refs/heads/master,refs/heads/beta,refs/heads/alpha',github.ref) && github.event_name == 'push' }}
-    steps:
-      - name: Checkout repo
-        uses: actions/checkout@v3
-      - name: Setup node
-        uses: actions/setup-node@v3
-        with:
-          cache: 'yarn'
-      - name: Install deps
-        # this runs a build script so there is no dedicated build
-        run: yarn install
-      - name: 🚀 Release
-        uses: cycjimmy/semantic-release-action@v3
-        with:
-          extra_plugins: |
-            @semantic-release/git
-          branches: |
-            [
-              'master',
-              {name: 'beta', prerelease: true},
-              {name: 'alpha', prerelease: true}
-            ]
+      - run: yarn install
+      - run: yarn build-storybook
+      # semantic-release skips not configured branches(see: release.config.js) or pull-requests
```

Comment:
we can securely execute `yarn release` here, because semantic-release will not make a release for PRs or not configured pushed-branches